docs/openapi/definitions/auth.yaml

```diff
@@ -0,0 +1,217 @@
+openapi: 3.0.1
+info:
+  title: Auth API
+  description:
+    Backstage passport-based auth-provider API. Each provider that is configured
+    in `@backstage/auth-backend` is supplied with the documented endpoints.
+  license:
+    name: Apache 2.0
+    url: http://www.apache.org/licenses/LICENSE-2.0.html
+  version: 0.1.1-alpha.7
+externalDocs:
+  description: Backstage official documentation
+  url: https://github.com/spotify/backstage/blob/master/docs/README.md
+servers:
+  - url: http://localhost:7000/auth/google
```

Comment:
```suggestion
  - url: http://localhost:7000/auth/google
  + url: http://localhost:7000/auth 
```

Reply:
accidental extra space at the end @soapraj :)